ngmix/moments.py

```diff
@@ -349,7 +349,7 @@ def g2mom(g1, g2, T):
     return Irr, Irc, Icc
 
 
-def make_mom_result(mom, mom_cov):
+def make_mom_result(mom, mom_cov, mom_norm=None):
```

Comment:
Is it true that for all the cases we use this function in `ngmix`, `mom` is actually `sums`?  In that case I think we should rename to `sums` for clarity

Reply:
Speaking for myself, this has caused me considerable confusion

Reply:
In that case, I think `mom_norm -> norm` or something

Reply:
I can rename to sums. Codes are gonna break for a while due to that. We should use `sums_norm` I think.